reporters/SponsorReporter.js

```diff
@@ -73,7 +73,7 @@ class SponsorReporter {
 
   async generateSponsorsTable() {
     await this.update();
-    console.log(italic("- All current token sponsors within the spesified EMP are printed"));
+    console.log(italic("- All current token sponsors within the specified EMP are printed"));
```

Comment:
@spellcheck Nick at it again. we should add this to our linter 😆

Reply:
lol we should definitely add an english language spell check to our linter

Reply:
+1